livechat/customer/web/client.py

```diff
@@ -835,3 +836,9 @@ def cancel_greeting(self,
 
 class CustomerWeb33(CustomerWebInterface):
     ''' Customer API version 3.3 class. '''
+    pass
```

Comment:
pass is not necessary (https://www.python.org/dev/peps/pep-0257/)

Reply:
👍